community/en-us/development/document.md

```diff
@@ -22,7 +22,9 @@ Make sure that your node version is 10+, docsite does not yet support versions h
 
 1. Run `npm install` in the root directory to install the dependencies.
 
-2. Run commands to collect resources 2.1. Run export PROTOCOL_MODE=ssh tells Git clone resource via SSH protocol instead of HTTPS protocol 2.2. Run ./scripts/prepare_docs.sh prepare all related resources, for more information you could see how prepare script work.
+2. Run commands to collect resources 
+   * Run `export PROTOCOL_MODE=ssh` tells Git clone resource via SSH protocol instead of HTTPS protocol.
+   * Run `./scripts/prepare_docs.sh` prepare all related resources, for more information you could see [how prepare script work](blob/master/HOW_PREPARE_WOKR.md).
```

Comment:
Modify the link to https://github.com/apache/dolphinscheduler-website/blob/master/HOW_PREPARE_WOKR.md